app/services/host_inventory_api.rb

```diff
@@ -56,7 +56,7 @@ def create_host_body
       'facts': [{ 'facts': { 'fqdn': @hostname }, 'namespace': 'inventory' }],
       'fqdn': @hostname,
       'display_name': @hostname,
-      'stale_timestamp': 1.week.from_now.to_s,
+      'stale_timestamp': 1.week.from_now.strftime('%FT%T%:z'),
```

Comment:
This works, but there's a shortcut method:

```suggestion
      'stale_timestamp': 1.week.from_now.iso8601,
```

The timezone is slightly different, it seems, but both conform to ISO 8601.